Jenkinsfile

```diff
@@ -0,0 +1,100 @@
+#!groovy
+
+node {
+ stage 'Checkout'
+ checkout scm
+
+ stage 'Docker Build and Push - Quay.io'
+ def quayUsername = "deis+jenkins"
+ def quayEmail = "deis+jenkins@deis.com"
+ withCredentials([[$class: 'StringBinding',
+                    credentialsId: '614c646f-57fc-4cdb-a70e-cbd5b0d91737',
```

Comment:
also note that this credential id no longer exists -- fixed in https://github.com/deis/docker-shell-dev/pull/2